doc/references/ImplementationOfVHDL.rst

```diff
@@ -118,19 +118,13 @@ Multiple standards can be used in a design:
 
 .. _psl_implementation:
 
-PSL implementation
+PSL support
 ==================
```

Comment:
Heading *underlines* need to have the same length as the text:

```
PSL support
===========
```

https://documentation-style-guide-sphinx.readthedocs.io/en/latest/style-guide.html#headings